ctn_waterloo/content/people/mariah-martin-shein.md

```diff
@@ -0,0 +1,15 @@
+name: Mariah Martin-Shein
+email: memartinshein@mta.ca
+one_liner: Visualization, Embodying Hierarchical Reinforcement Learning
+group: Grad students
+title: Masters Student
+picture: /static/img/placeholder.png
+links:
+    - text: CV (March 2014)
+      url: /files/people/aubin-cv.pdf
+    - text: Github
+      url: https://github.com/Seanny123
+    - text: Twitter
+      url: https://twitter.com/voyageur_techno
```

Comment:
@Seanny123 : this is all your info. Also, I'd let people add their own website page when they get here. That way, they learn how it works!